src/main/resources/react4xp/_entries/ArticleArchive.jsx

```diff
@@ -0,0 +1,170 @@
+    listOfArticlesSectionTitle,
+    showMore,
+    showMorePagination
+  } = props
+
+  const [articles, setArticles] = useState(firstArticles.articles)
+  const [totalCount, setTotalCount] = useState(firstArticles.total)
+  const [loading, setLoading] = useState()
+
+  function fetchArticles() {
+    setLoading(true)
+    get(articleArchiveService, {
+      params: {
+        start: articles.length,
+        count: 15,
+        language: language,
+        pageId: pageId
+      }
+    }).then((res) => {
+      if (res.data.articles.length) {
+        setArticles((prev) => [...prev, ...res.data.articles])
+        setTotalCount(res.data.total)
+      }
+    })
+      .finally(() => {
+        setLoading(false)
+      })
+  }
+
+  function renderShowMoreButton() {
+    if (totalCount > articles.length) {
+      return (
+        <Row className="justify-content-center">
+          <Col className="col-auto">
+            <Button onClick={() => fetchArticles()}>
+              <span className="sr-only">{`${showMorePagination.replace('{0}', articles.length)} ${totalCount}`}</span>
+              {!loading ?
+                <span aria-hidden="true">
+                  <ChevronDown size="18" className="chevron-icons" />{`${showMore} ${articles.length} / ${totalCount}`}
+                </span> :
+                <span aria-hidden="true" className="spinner-border spinner-border-sm" />
+              }
+            </Button>
+          </Col>
+        </Row>
+      )
+    }
+    return null
+  }
+
+  function addArticles() {
+    if (articles.length) {
+      const groupByYear = groupBy((articles) => {
+        return articles.year
+      })
+      const groupArticlesByYearDesc = Object.entries(groupByYear(articles)).reverse()
+      return groupArticlesByYearDesc.map(([year, articles], index) => {
+        return (
+          <Row className="articles-container" key={`groupedArticles-${index}`}>
+            <Col className="col-12 col-lg-1 p-0">
+              <h2 id={`archive-articles-${year}`}>{year}</h2>
+            </Col>
+            <Col id="article-archive-list" className="col-12 col-lg-8 p-0">
+              <ol className="row p-0" aria-labelledby={`article-archive-heading archive-articles-${year}`}>
+                {articles.map((article, index) => {
+                  const srSubtitle = article.subtitle.replace(' / ', ' ')
+                  return (
+                    <li key={`articles-${year}-${index}`} className="article-container col-12 p-0 mb-5">
+                      <Link linkType="header" href={article.href}>
+                        <span aria-hidden="true">{article.title}</span>
+                        <span className="sr-only">
+                          {`${article.title} ${srSubtitle.replace(year, '')}`}
+                        </span>
+                      </Link>
+                      <Paragraph className="mt-2 mb-1">{article.preamble}</Paragraph>
+                      <Text small>{article.subtitle}</Text>
```

Comment:
Maybe hide this with aria-hidden since subtitle is added to Span with class sr-only?